src/Components/Navigation/index.js

```diff
@@ -1,24 +1,26 @@
 import React, { useState } from "react";
 
-const Navigation = ({ searchByKeyWord, categories }) => {
+const Navigation = ({ searchByKeyWord, categories, filterByCategory }) => {
   const [keyWord, setKeyWord] = useState('');
-  const [selectCategory, setSelectCategory] = useState('');
-  
+
   const searchPostings = (e) => {
     e.preventDefault();
     searchByKeyWord(keyWord)
     setKeyWord('');
   }
-  
+
   return(
     <section>
       <input type="text" value={keyWord} placeholder="i.e Boulder..." onChange={(e) => setKeyWord(e.target.value)}></input>
       <input type="submit" value="search" onClick={(searchPostings)}></input>
       <article>
-        <select name="category" >
-          <option selected value>-- select type --</option>
+        <select name="category" onChange={(e) => filterByCategory(e.target.value)}>
```

Comment:
I like the drop down menu for filter method. This way it makes it easier for the user to choose a specific category instead of trying to find out what categories we have by typing keywords